docs/source/1.0/spec/aws/customizations/s3-customizations.rst

```diff
@@ -0,0 +1,132 @@
+========================
+Amazon S3 Customizations
+========================
+
+.. contents:: Table of contents
+    :depth: 2
+    :local:
+    :backlinks: none
+
+S3 Bucket Addressing
+====================
+
+Clients for Amazon S3 SHOULD expose multiple levels of configuration for bucket
```

Comment:
We should move some of this language out to a general thing describing the config. Not really important for now though